### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by Robison-faike123**
For the first milestone in the process of creating my portfolio website, I will:

- [ ] set up a new repository and prepare it for development using best practices (e.g. linters).
- [ ] build the first 2 sections of the mobile website using the template you chose in the previous step.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <div class="container">
+      <div class="AppBar">
+        <header class="ToolBar">
+          <nav>
+            <img src="images/NormalButton.png" alt="" class="NormalButton" />
+          </nav>
+          <div class="Logo">
+            <a href="#">faishar faike</a>
+          </div>
+        </header>
```
Comment (awais-amjed) [inline index.html]: It's better to align this in the middle and also maybe decrease the height a bit to follow the Figma design 🤗

Your design:

image

Figma design:

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  </head>
+  <body>
+    <div class="container">
+      <div class="AppBar">
+        <header class="ToolBar">
+          <nav>
+            <img src="images/NormalButton.png" alt="" class="NormalButton" />
+          </nav>
+          <div class="Logo">
+            <a href="#">faishar faike</a>
```
Comment (awais-amjed) [inline index.html]: Kindly remove the underline from the Logo and increase its font-size to match the Figma design 🤗

File: index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          </div>
+        </header>
+      </div>
+      <div class="headline">
+        <div class="content">
+          <div class="PrimaryText">
+            <h1 class="PrimaryText">Hey There. I'm faishar</h1>
```
Comment (awais-amjed) [inline index.html:28]: Kindly use the Font-family, font-size and weight from the Figma design 🤗

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+          </div>
+          <div class="SupportingText">
+            <p class="SupportingText">I'm a Software Developer</p>
```
Comment (awais-amjed) [inline index.html]: Kindly use the Font-family, font-size, and weight from the Figma design 🤗

image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+          </div>
+          <div class="discription">
+            <p class="discription">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don't hestiate to contact me.
+            </p>
```
Comment (awais-amjed) [inline index.html:38]: Kindly use the Font-family, font-size, and weight from the Figma design 🤗

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+        <header class="ToolBar">
+          <nav>
+            <img src="images/NormalButton.png" alt="" class="NormalButton" />
+          </nav>
+          <div class="Logo">
+            <a href="#">faishar faike</a>
+          </div>
+        </header>
+      </div>
+      <div class="headline">
```
Comment (awais-amjed) [inline index.html]: Kindly replace this `div` with a `section` tag 🤗

- A section represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

File: index.html (hunk 4 above)
Comment (awais-amjed) [inline index.html]: Kindly make sure that the subtitle uses a header tag `hx (examples: <h2>,<h3>, etc...)` 🤗

- This way will be semantically correct and the screen readers and other technologies will thank you.

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @Robison-faike123 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - The linters checks are failing. Please make sure you fix all linters before submitting a PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Comment by Robison-faike123** [comment]
I have worked on it and it has worked .I have added google font family link
and also put sans serif  as default font family

**Pull request merged**